tests/common/_docker_hub_registry_mirror.yml

```diff
@@ -12,7 +12,7 @@ containerd_registries:
 
 containerd_max_container_log_line_size: -1
 
-crio_registries_mirrors:
+crio_registries:
   - prefix: docker.io
     insecure: false
     blocked: false
```

Comment:
packet_almalinux8-crio job was failed like
```
Wednesday 21 December 2022  12:20:37 +0000 (0:00:00.904)       0:03:24.503 **** 
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible.errors.AnsibleUndefinedVariable: 'dict object' has no attribute 'unqualified'
fatal: [instance-2]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'unqualified'"}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible.errors.AnsibleUndefinedVariable: 'dict object' has no attribute 'unqualified'
fatal: [instance-1]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'unqualified'"}
```
It is necessary to add `unqualified: false` here to solve this issue.

Reply:
/lgtm cancel

Reply:
Cool, thanks for updating.